src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -333,17 +278,14 @@ public int compare(final ProTerritory t1, final ProTerritory t2) {
     });
 
     // Log prioritized territories
-    for (final ProTerritory patd : prioritizedTerritories) {
+    for (final ProTerritory patd : attackOptions) {
```

Comment:
Second comment and/or alternative, consider having ProLogger to own a first class debugging function, ie: `ProLogger.debug( attackOptions )`. That would be better IMO since you would then have better adherence to SRP.  The pro debugger would be responsible for the formatting and the filler text, while this method is just responsible for passing the `attackOptions` to the debugger.  That is nicer than this class doing stuff with attackOptions, then going down a level of abstraction to then formatting a debug output line and then sending raw debug output.